src/scraper/scraper.cpp

```diff
@@ -477,13 +468,11 @@ class stringbuilder
 /*********************
 * Whitelist Data     *
 *********************/
-// TODO: Lock this somehow
 int64_t SuperblockAge()
 {
-    int64_t superblock_time = ReadCache(Section::SUPERBLOCK, "magnitudes").timestamp;
-    int64_t nSBage = GetAdjustedTime() - superblock_time;
+    LOCK(cs_main);
```

Comment:
I didn't see any places where a call to this function is nested after another lock. The conservative lock-scoping strategy you took with the scraper guarantees that no locks are active on the scraper threads when this function is called. It may be better to move the implicit lock out of this function, though.

Reply:
I agree with your comments. I think it is ok like it is!